DOCKER/docker-entrypoint.sh

```diff
@@ -15,6 +15,7 @@ if [ ! -d "$TMHOME/config" ]; then
 			"$TMHOME/config/config.toml" &&
 		sed -i \
 			-e "s/^\s*\"chain_id\":.*/  \"chain_id\": \"$CHAIN_ID\",/" \
+			-e "s/^\s*\"time_iota_ms\":.*/  \"time_iota_ms\": \"500\"/" \
 			"$TMHOME/config/genesis.json"
```

Comment:
Shouldn't we prefer `jq` to update a genesis file? This script will break as soon as time_iota_ms is not the last object entry anymore because of the missing trailing comma.

Reply:
I was hoping no-one would notice. 😉 Updated now, thanks for the nudge!

Reply:
👨‍🏫